src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -77,24 +82,26 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
     return (
         <>
             <List className={classes.list}>
-                <ListItem key={'NodeEditorModificationsCount'}>
-                    <FormattedMessage
-                        id={'network_modification/modificationsCount'}
-                        values={{ count: modifications?.length }}
-                    />
-                </ListItem>
+                <ListItemText
+                    className={classes.modificationCount}
+                    primary={intl.formatMessage(
+                        { id: 'network_modification/modificationsCount' },
+                        { count: modifications?.length }
+                    )}
+                />
                 <Divider />
                 {modifications?.map((item) => (
                     <ModificationListItem key={item.uuid} modification={item} />
                 ))}
             </List>
 
-            <Button
+            <Fab
                 className={classes.addButton}
+                color="primary"
```

Comment:
color should adapt from the dark or light theme, not sure you must set it here.

Reply:
it does.
https://v4.mui.com/components/floating-action-button/

Reply:
my bad